microcosm_pubsub/envelope.py

```diff
@@ -69,6 +70,18 @@ def parse_media_type_and_content(self, message):
         return media_type, loads(content)
 
 
+class NaiveMediaTypeAndContentParser(MediaTypeAndContentParser):
```

Comment:
It's the difference between having your input file be:

```
{"mediaType": ..., "uri": ...}
```

And:
```
{"Body":"{\"mediaType\": ..., \"uri\": ...}"}
```

Or something equally awful.

It also lets us default some of the SQS required parameters (message id, receipt handle) that are irrelevant in this scenario.